catalog/build.gradle.kts

```diff
@@ -10,12 +10,17 @@ android {
 
 dependencies {
 
+    implementation(libs.kotlinxCoroutinesCore)
+    implementation(libs.kotlin.reflect)
```

Comment:
Do we really need reflection? :/

Reply:
Nope, omission on my end. Thanks for catching!

Reply:
removed